helm/hpcc/templates/eclccserver.yaml

```diff
@@ -14,7 +14,7 @@ data:
 {{- include "hpcc.generateLoggingConfig" . | indent 6 }}
       queues:
 {{ include "hpcc.generateConfigMapQueues" .root | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "codeVerify") ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .secretsCategories ) | indent 6 }}
```

Comment:
probably hpcc.*configMap* func comment needs updating to say "secretsCategories" now expected/passed in.